app/containers/TokenSale/TokenSale.jsx

```diff
@@ -90,7 +90,20 @@ class TokenSale extends Component<Props, State> {
     }
   }
 
+  componentDidUpdate(prevProps: Props) {
+    if (
+      prevProps.assetBalances !== this.props.assetBalances &&
```

Comment:
I think it's better to use `lodash` isEqual for comparison.

Reply:
👍